### ISSUE TITLE
ctype argument error with Python3

### ISSUE THREAD

**Issue by ubutnux** [stevemarple/python-zwoasi#5]
Get an error when using set_roi function
```
self.camera.set_roi(bins=1)
  File "C:\Users\ucup\Anaconda3\envs\opencv\lib\site-packages\zwoasi\__init__.py", line 449, in set_roi
    self.set_roi_format(width, height, bins, image_type)
  File "C:\Users\ucup\Anaconda3\envs\opencv\lib\site-packages\zwoasi\__init__.py", line 379, in set_roi_format
    _set_roi_format(self.id, width, height, bins, image_type)
  File "C:\Users\ucup\Anaconda3\envs\opencv\lib\site-packages\zwoasi\__init__.py", line 120, in _set_roi_format
    r = zwolib.ASISetROIFormat(id_, width, height, bins, image_type)
ctypes.ArgumentError: argument 2: <class 'TypeError'>: wrong type

```

**Comment by stevemarple** [comment]
I think the problem comes from using Python 3 where division results in a float not an int. I believe I have fixed the code but I don't have a spare camera to hand to test it with. Please update to the latest version and let me know if the issue is resolved.
Steve

**Comment by ubutnux** [comment]
yes. you are right. resolved

**Comment by stevemarple** [comment]
Thanks for letting me know.
Fixed by 7dc5b297473be6c8648d62ec3902d5b1b5653bf3.
